### ISSUE TITLE
Emoji not working with NetworkFileTranslationLoader but works with FileTranslationLoader

### ISSUE THREAD

**Issue by chamartt** [ilteoood/flutter_i18n#190]
Hello,

In my json file, I have some Emojis as 🤝.
When I use `FileTranslationLoader`, emojis are correclty displayed on my app, but when I use `NetworkFileTranslationLoader` (with same source json files), emojis are converted  as specials caracters.

I tried to add `decodeStrategies: [JsonDecodeStrategy(), YamlDecodeStrategy(), XmlDecodeStrategy(), TomlDecodeStrategy()]` in parameter of my `NetworkFileTranslationLoader` but nothing changed.

This is the (good) result with `FileTranslationLoader`:
![SmartSelect_20220316-124411](https://user-images.githubusercontent.com/15814545/158536541-0e080a33-b097-42a6-a7a2-c141f658f2b4.jpg)

This is the (wrong) result with `NetworkFileTranslationLoader`:
![SmartSelect_20220316-124033](https://user-images.githubusercontent.com/15814545/158536595-de0e8193-fb48-4417-8e1a-7d4997fc510a.jpg)

I'm using `flutter_i18n` 0.32.1, `Flutter sdk` 2.10.3 and `Dart` 2.16.1

Thank you for your work and your time to answer.

**Comment by ilteoood** [comment]
Hi, do you have a direct link where I can try it?

**Comment by chamartt** [comment]
Hello,

This is my code implementation :
```
NetworkFileTranslationLoader(
           useCountryCode: false,
           fallbackFile: 'en',
           baseUri: Uri.https("l-algo-de-paulo-8fb41.appspot.com.storage.googleapis.com", "translations"),
         )
```

I have one `en.json` and `fr.json` in the translations folder.

Thank you!

**Comment by chamartt** [comment]
Hi @ilteoood,

Do you need additionnals informations to reproduce ?
We are waiting this point to upgrade our application.

Let me know if you need something else !

Thank you for your work and your time.

**Comment by ilteoood** [comment]
This should be fixed with the release 0.32.2

One suggestion: use the `decodeStrategies` parameter in order to avoid useless network calls.

P.S. this is an open source project, instead of ping me for a solution, you can contribute :smile:

**Comment by chamartt** [comment]
Hi,

Thank you for the MPR !

Unfortunatly I don't know as you how your plugin works and I had lot of work to look what was the problem, maybe was just one configuration in my call but I'm "happy" to see that was worst ! 

Thank you also for the suggestion, I tried with all to look results but I will restrict only JSON files !
